examples/film/film_trainer.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import os
+
+sys.path.insert(0, os.path.abspath('../../'))  # adds path2gammagl to execute in command line.
+import argparse
+import tensorlayerx as tlx
+from tensorlayerx.model import TrainOneStep, WithLoss
+from gammagl.datasets.ppi import PPI
+from gammagl.models import FILMModel
+from gammagl.loader import DataLoader
+
+batch_size = 2
```

Comment:
I think we may put this into ``parser.add_argument`` instead of a global variable.